### PR TITLE
Add staging environment

### PR DESCRIPTION
**Pull request by 0xGabi**
TODO:

- [ ] Deploy registries on Staging

### CODE REVIEW

File: arapp.json [hunk 1]
```diff
@@ -22,11 +22,23 @@
...
     "mainnet": {
       "registry": "0x314159265dd8dbb310642f98f50c066173c1259b",
       "appName": "placeholder-app-name.open.aragonpm.eth",
       "wsRPC": "wss://mainnet.eth.aragon.network/ws",
       "network": "mainnet"
+    },
+    "ropsten": {
+      "registry": "0x6afe2cacee211ea9179992f89dc61ff25c61e923",
+      "appName": "placeholder-app-name.open.aragonpm.eth",
+      "wsRPC": "wss://ropsten.infura.io/ws",
```
Comment (0xGabi) [inline arapp.json]: @sohkai Is there a Ropsten node?

File: arapp.json [hunk 2]
```diff
@@ -22,11 +22,23 @@
       "wsRPC": "wss://rinkeby.eth.aragon.network/ws",
       "network": "rinkeby"
     },
+    "staging": {
+      "registry": "0x98df287b6c145399aaa709692c8d308357bc085d",
```
Comment (ajsantander) [inline arapp.json]: Double checking this address, I get a different one. I looked here: https://github.com/aragon/deployments/blob/master/environments/rinkeby/readme.md

File: arapp.json [hunk 3]
```diff
@@ -22,11 +22,23 @@
...
+      "network": "rinkeby"
+    },
     "mainnet": {
       "registry": "0x314159265dd8dbb310642f98f50c066173c1259b",
       "appName": "placeholder-app-name.open.aragonpm.eth",
       "wsRPC": "wss://mainnet.eth.aragon.network/ws",
       "network": "mainnet"
+    },
+    "ropsten": {
+      "registry": "0x6afe2cacee211ea9179992f89dc61ff25c61e923",
```
Comment (ajsantander) [inline arapp.json]: As above, https://github.com/aragon/deployments/blob/master/environments/ropsten/readme.md

File: arapp.json (hunk 2 above)
Comment (0xGabi) [reply arapp.json]: I choose the same as Rinkeby cause this: https://github.com/aragon/deployments/issues/54

File: arapp.json (hunk 3 above)
Comment (0xGabi) [reply arapp.json]: This one seems right 🤔

**Comment by 0xGabi** [comment]
Perfect, make sense. I'll remove Ropsten.

In my experience so far seems that having a staging environment is useful as a step in the development workflow specially right now that most of the user testing is been done on Rinkeby

**Pull request merged**